config/platform.php

```diff
@@ -193,4 +193,19 @@
         'cache' => false
     ],
 
+    /*
+    |--------------------------------------------------------------------------
+    | Routes
+    |--------------------------------------------------------------------------
+    |
+    | Default configuration for platform routes
```

Comment:
What do you think about adding such a description?

> If the request does not match any route and arguments, Orchid will automatically generate its own 404 page. It can be disabled if you want to declare routes on the same domain and prefix or create your own page.

Reply:
Super!